src/test/java/hw2/ex1/FirstScenarioTest.java

```diff
@@ -0,0 +1,203 @@
+        assertEquals(actualUserName.getText(), "PITER CHAILOVSKII");
+
+        // 5. Assert Browser title
+        // There's a getting of a title and
+        // comparing it with expected title -- "Home page"
+        // NB! We take expectedTitle from above
+        actualTitle = driver.getTitle();
+        assertEquals(actualTitle, "Home Page");
+
+        // 6. Assert that there are 4 items on the header
+        // section are displayed and they have proper texts
+        // There's a creating of array list with expected texts
+        // in header, getting a web element with actual header
+        // texts, getting list of these texts and comparing them
+        ArrayList<String> expectedHeaderTexts = new ArrayList<>();
+        expectedHeaderTexts.add("HOME");
+        expectedHeaderTexts.add("CONTACT FORM");
+        expectedHeaderTexts.add("SERVICE");
+        expectedHeaderTexts.add("METALS & COLORS");
+        WebElement header = driver.findElement(By.
+                xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']"));
+        List<String> actualHeaderTexts = Arrays.
+                asList(header.getText().split("\n"));
+        assertEquals(actualHeaderTexts, expectedHeaderTexts);
+
+        // 7. Assert that there are 4 images on the Index Page and
+        // they are displayed
+        // There's a simple assertion of existing
+        // and displaying of 4 icons
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-practise']")).
+                isDisplayed());
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-custom']")).
+                isDisplayed());
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-multi']")).
+                isDisplayed());
+        assertTrue(driver.findElement(By.
+                xpath("//span[@class='icons-benefit icon-base']")).
+                isDisplayed());
+
+        // 8. Assert that there are 4 texts on the Index Page
+        // under icons and they have proper text
+        // There's a creating of expected benefits texts list and
+        // filling it with values. Then we get the list of elements
+        // with actual benefits texts, get texts from them
+        // and compare actual texts and expected ones
+        ArrayList<String> expectedBenefitsTexts = new ArrayList<>();
+        expectedBenefitsTexts.add("To include good practices\n" +
+                "and ideas from successful\n" +
+                "EPAM project");
+        expectedBenefitsTexts.add("To be flexible and\n" +
+                "customizable");
+        expectedBenefitsTexts.add("To be multiplatform");
+        expectedBenefitsTexts.add("Already have good base\n" +
+                "(about 20 internal and\n" +
+                "some external projects),\n" +
+                "wish to get more…");
```

Comment:
Creation of array could be simplified

Reply:
Solved: I changed *add*-method on *Arrays.asList(. . .)*